integration/tutorial/iogroup/ExampleIOGroup.cpp

```diff
@@ -455,7 +483,11 @@ int ExampleIOGroup::signal_behavior(const std::string &signal_name) const
                         GEOPM_ERROR_INVALID, __FILE__, __LINE__);
     }
     // All example signals are time based and increase monotonically
-    return M_SIGNAL_BEHAVIOR_MONOTONE;
+    int result = M_SIGNAL_BEHAVIOR_MONOTONE;
+    if (signal_name != "TMP_FILE_CONTROL" && signal_name != "EXAMPLE::TMP_FILE_CONTROL") {
+        result = M_SIGNAL_BEHAVIOR_VARIABLE;
+    }
```

Comment:
Is this the opposite of the intended behavior? I think the control file is the only variable one.